earthworks-aardvark-stage/synonyms.txt

```diff
@@ -0,0 +1,389 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#-----------------------------------------------------------------------
+#some test synonym mappings unlikely to appear in real input text
+aaa => aaaa
+bbb => bbbb1 bbbb2
+ccc => cccc1,cccc2
+a\=>a => b\=>b
+a\,a => b\,b
+fooaaa,baraaa,bazaaa
+#The OGP file also includes these mappings
+aaafoo => aaabar
+bbbfoo => bbbfoo bbbbar
+cccfoo => cccbar cccbaz
+
+# Some synonym groups specific to this example
+GB,gib,gigabyte,gigabytes
+MB,mib,megabyte,megabytes
+Television, Televisions, TV, TVs
+#notice we use "gib" instead of "GiB" so any WordDelimiterFilter coming
+#after us won't split it into two words.
+
+# Synonym mappings can be used for spelling correction too
+pixima => pixma
+
+# Copied from OGP LCSH synonyms
+Harbors,Pier,Quay,Warf
+
+Hydrocarbons,CNG,Coal,ethanol,gas,gasoline,kerosene,LNG,oil,OPEC,petrol,Petroleum,propane
+
+Walls,Fences,Fortification,fortresses,Gates,Hedges,Seawall,Wall,Walls
+
+Roads,arteries,automobiles,Avenue,Avenues,beltways,Boulevards,driveways,expressway,expressways,Freeways,highway,highways,intersections,interstate,interstates,lanes,motorways,road,Roadblocks,Roads,Roadways,route,Routes,Speedlimits,Street,Streets,Thoroughfares,Tollbooths,way
+
+Schools,academies,College,colleges,colleges/universities,Education,preschool,preschools,school,Schools,universities,university,University/Colleges
+
+Pedestrians,Curbs,Roads,Footpaths,handicap,Pavements,Pedestrian,sidewalk,Sidewalks,walk,Walking,Walkways
+
+Atmosphere,Air,Atmosphere,climate,Climatology,climatologymeteorology,Cloudiness,Clouds,Evapotrans,Evapotranspiration,haze,humidity,hurricane,Isohyets,Meteorological,Meteorology,Precipitation,Rain,rainfall,Sunshine,temperature,weather,wind
+
+Altitudes,Altitude,Altitudes,Aspect,bathymetry,breaklines,contour,Contours,DEM,depths,DTM,echosounder,elevations,GTOPO30,highpoint,hillshaded,Hypsography,ruggedness,SRTM,terrain,topographic,topography,Topoquads
+
+History,ancestry,Historic,historical,History,Memorial,Memorials,Ruins
+
+Industries,chemists,Factories,Buildings,Factory,Buildings,industrial,industries,industry,Iron-works,Manufacture,refineries
+
+Water-supply,Drinking,Water-pipes,Water-supply,Waterworks
+
+watersheds,catchments,hydrology,drainage,hydrology,drainages,hydrology,watershed,hydrology
+
+fishes,anadromous,fish,fisheries,fishing,oncorhynchus,spawn,
+
+census,ageb,cbsa,census
+
+mountains,hill,hills,mount,mountain,mountains,peak,peaks,ridge,ridgelines,geology,summit
+
+bicycles,bicycle,bike
+
+sewerage,drains,manhole,manholes,sewage,sewer,sewerage,sewer-pipe,sewers
+
+invertebrates,lobsters,mosquitoes,shellfish
```

Comment:
@jcoyne and @peetucket , I can change this line so it's an expansion in one direction. I.e. invertebrates -> lobsters, mosquitoes, shellfish.  This would prevent a search for mosquitoes to expand to a search for lobsters.

Reply:
LOL, I don't really care. It's not my area of expertise. I just thought that it was silly.

Reply:
"The Flying Lobsters" is going to be the name of my steel drum band when I retire on Cape Cod.